### PR TITLE
Change parent in `octavia-proxy-build-image` job

### PR DESCRIPTION
**Pull request by lego963**

### CODE REVIEW

File: .zuul.yaml [hunk 1]
```diff
@@ -25,6 +25,7 @@
             &imagetag "{{ zuul.tag is defined | ternary([zuul.get('tag', '').split('.')[0], '.'.join(zuul.get('tag', '').split('.')[:2]), zuul.get('tag', '')], ['latest']) }}"
           siblings:
             - github.com/opentelekomcloud/python-otcextensions
+    nodeset: ubuntu-focal
```
Comment (gtema) [inline .zuul.yaml:28]: I would honestly rather set nodeset for build-docker-image instead. We have same issue everywhere

**Review by gtema — CHANGES_REQUESTED** [review]

File: .zuul.yaml (hunk 1 above)
Comment (lego963) [reply .zuul.yaml:28]: `build-docker-image` job came from opendev

File: .zuul.yaml [hunk 2]
```diff
@@ -34,6 +35,7 @@
       - name: opentelekomcloud/python-otcextensions
         override-checkout: master
     vars: *octavia-proxy_image_vars
+    nodeset: ubuntu-focal
```
[the remark this answers is not in the document]
Comment (lego963) [reply .zuul.yaml:38]: parent job doesn't have concrete `nodeset`

File: .zuul.yaml (hunk 1 above)
Comment (gtema) [reply .zuul.yaml:28]: then we should define something like otc-build-docker-image in otc-zuul-jobs. I anyway wanted to also experiment with build-container-image instead and also trying to use buildah. This should be possible to achieve inside pod

**Review by gtema — APPROVED** [review]

**Pull request merged**